corehq/apps/formplayer_api/smsforms/tests/test_formplayer_interface.py

```diff
@@ -0,0 +1,67 @@
+import uuid
+
+import requests_mock
+from django.conf import settings
+from django.test import TestCase
+
+from corehq.apps.formplayer_api.smsforms.api import FormplayerInterface
+from corehq.apps.formplayer_api.utils import get_formplayer_url
+from corehq.util.hmac_request import get_hmac_digest
+
+
+class TestFormplayerInterface(TestCase):
+    def setUp(self):
+        super().setUp()
+        self.session_id = uuid.uuid4().hex
+        self.domain = uuid.uuid4().hex
+        self.user_id = uuid.uuid4().hex
+        self.interface = FormplayerInterface(self.session_id, self.domain, self.user_id)
+
+    def test_get_raw_instance(self):
+        expected_request_data = {
+            'action': 'get-instance',
+            'session-id': self.session_id,
+            'session_id': self.session_id,
+            'domain': self.domain,
+            'oneQuestionPerScreen': True,
+            'nav_mode': 'prompt',
+        }
+        self._test_request("get-instance", self.interface.get_raw_instance, expected_request_data, {})
+
+    def test_answer_question(self):
+        mock_response = {"event": {
+            "datatype": "select",
+            "choices": ["red", "green", "blue"],
+            "caption": "What's your favorite color?",
+            "type": "question",
+            "answer": None,
+            "required": 0,
+            "ix": "1",
+            "help": None
+        }}
+
+        expected_request_data = {
+            'action': 'answer',
+            'answer': 'answer1',
+            'session-id': self.session_id,
+            'session_id': self.session_id,
+            'domain': self.domain,
+            'oneQuestionPerScreen': True,
+            'nav_mode': 'prompt',
+        }
+
+        def request_callable():
```

Comment:
calling the parameter `request_callable` makes sense, because it has to be generic. The name of this function, however, doesn't need to be generic -- `answer_question` is more in line with the previous test's `get_raw_instance`